File: framework/ecommerce-framework-api/src/main/java/com/sdl/ecommerce/api/model/ProductVariantAttribute.java
package com.sdl.ecommerce.api.model;

/**
 * Product Variant Attribute
 *
 * @author nic
 */
public interface ProductVariantAttribute {

    /**
     * @return id of the variant attribute
     */
    String getId();

    /**
     * @return name of the variant attribute (can be localized)
     */
    String getName();

    /**
     * @return value identity
     */
    String getValueId();

    /**
     * @return presentable value (can be localized)
     */
    String getValue();
}

File: dxa.net/ecommerce-framework-dotnet/SDL.ECommerce.Rest/Model/ProductVariantAttribute.cs
﻿using SDL.ECommerce.Api.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SDL.ECommerce.Rest.Model
{
    public class ProductVariantAttribute : IProductVariantAttribute
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Value { get; set; }
        public string ValueId { get; set; }
    }
}
